### PR TITLE
Feature/small fixes and updates

### PR DESCRIPTION
**Pull request by hrwakeford**
Adding information to the output PDF and fixing the positive AIC condition

### CODE REVIEW

**Comment by hrwakeford** [comment]
@ivalaginja I think that we need to inflate the input errors before the second fit by multiplying by the beta value when it is a valid number. How would we do this?

**Comment by hrwakeford** [comment]
To outline the changes: 
I have reset the positive AIC condition to just be an extremely low number. This will allow users to fit any dataset even if it cannot be well fit by and of the systematic models in the grid. I then modified the output PDF to give the DOF, Chi-squared, and the AIC evidence values for the top models along with warnings to the user to check the values for valid results.

File: exoticism/marginalisation.py [hunk 1]
```diff
@@ -412,7 +412,7 @@ def total_marg(exoplanet, x, y, err, sh, wavelength, output_dir, run_name, plott
     print('AIC for all systems: {}'.format(sys_evidenceAIC))
 
     # REFORMAT all arrays, masking all negative AIC values
```
Comment (ivalaginja) [inline exoticism/marginalisation.py:414]: I would suggest to also change this comment to reflect the changes in the line below (negative AIC --> poor AIC evidence).

Comment (hrwakeford) [reply exoticism/marginalisation.py:414]: Good idea I will do that

**Comment by ivalaginja** [comment]
@hrwakeford is your question about inflating the errors with the beta value still relevant? I can look into it if you want.

**Comment by hrwakeford** [comment]
I was thinking about it and I was not sure how to test it out. I want to see if inflating the input err on the data points after the first fit with the beta value from the noise calculator would be something useful. But was not sure how to test that.

**Comment by hrwakeford** [comment]
You would need to add the following into the first fit loop to calculate `beta`
```
        systematic_model = marg.sys_model(phase, HSTphase, sh, tmodel.m_fac.val, tmodel.hstp1.val, tmodel.hstp2.val,
                                          tmodel.hstp3.val, tmodel.hstp4.val, tmodel.xshift1.val, tmodel.xshift2.val,
                                          tmodel.xshift3.val, tmodel.xshift4.val)

        fit_model = mulimb01 * tmodel.flux0.val * systematic_model     #  Issue #36
        residuals = (img_flux - fit_model) / tmodel.flux0.val

        white_noise, red_noise, beta = marg.noise_calculator(residuals)
```

Then save the beta in an array for each systematic model and then use them individually to inflate the light curve uncertainty for that systematic model fit.

**Comment by hrwakeford** [comment]
I was not sure how to do the last part, save it and then tell the fit to use the new_err

**Comment by ivalaginja** [comment]
I remember we had investigated the scaling of the errors between fits before, likely in one of the notebooks and/or tests when we were checking the correct error propagation when we were migrating to Python. I will try to dig that out and reconstruct it, it will just take me a little bit to remind myself of the structure of the data being passed around. In any case, if we managed to do it once I am confident we can do it again.

**Comment by hrwakeford** [comment]
@ivalaginja I am going to separate this into two requests so that we can get the little fixes above in the code. I will then open an issue on the beta_noise error inflation and we can work on that separately. 
Does that sound reasonable?

Note I need to rebase this before anything else can be merged.

**Comment by ivalaginja** [comment]
Sounds perfect!

**Comment by hrwakeford** [comment]
The enhancements with 'beta' are now in issue #117

**Review by ivalaginja — APPROVED** [review]
@hrwakeford I wasn't sure whether you wanted to add anything else to this PR, now that the other issue has been exported? If not, this is good to merge.

**Pull request merged**